docs/insomnia/migrate-from-designer.md

```diff
@@ -35,28 +39,44 @@ If there is an unexpected error during migration, you will be shown the error th
 
 ## Manual restore
 
-You may also choose to manually restore from the backup directory created at <app-data-dir>/insomnia/core-backup. In the current state, each time you migrate, this backup is overwritten just prior to starting migration. Where is app-data-dir?
+You may also choose to manually restore from the backup directory created at `<app-data-dir>/insomnia/core-backup`. In the current state, each time you migrate, this backup is overwritten just prior to starting migration.
 
-The <app-data-dir>/insomnia/core-backup directory contains the following data:
+See [Backup](#backup) for a list of `app-data-dir` locations based on your OS.
 
-* insomnia.{model}.db database files
+The `<app-data-dir>/insomnia/core-backup` directory contains the following data:
+
+* `insomnia.{model}.db` database files
 * plugins folder
 * responses folder
 * version-control folder
 
-Each of these items also exists in the parent folder, <app-data-dir>/insomnia, which contains the "current" data used by the application. 
+Each of these items also exists in the parent folder, `<app-data-dir>/insomnia`, which contains the current data used by the application.
 
-In order to manually restore:
+In order to manually restore, do the following:
 
 1. Close Insomnia
-2. Delete the plugins, responses, and version-control folders in the <app-data-dir>/insomnia live data directory
-3. Copy-paste everything from <app-data-dir>/insomnia/core-backup into <app-data-dir>/insomnia, overwriting everything that exists
+2. Delete the plugins, responses, and version-control folders in the `<app-data-dir>/insomnia` live data directory
+3. Copy-paste everything from `<app-data-dir>/insomnia/core-backup` into `<app-data-dir>/insomnia`, overwriting everything that exists
 4. Launch Insomnia
 
-## Edge cases
+## Edge case
+
+If you moved your workspace from Core to Designer several months ago, and are now merging the same workspace (with the same id) from Designer back into Core, this edge case applies. The merge won't remove any children of the workspace (such as requests, request groups, certificates), but if a duplicate id is found, it will favor the data coming from Designer.
+
+In the unlikely scenario that you can't see your requests or workspace in Insomnia after migration, [restore your data from your backup](#restore-from-backup). Alternatively, because the migration is primarily additive and will not remove anything from the existing database (it will overwrite if the same id is found), some entries may become orphaned. Find orphaned entries by opening the `insomnia.{model}.db` files.
+
+If you have orphaned entries, or are not seeing data you expect to see when you launch Insomnia, send an email to [Support](mailto:support@insomnia.rest) outlining your steps, including your application logs, and the team will assist where possible.
+
+## Migration Alternatives
```

Comment:
Should this section be the first section? It seems like the most important information, now that the only way to migrate is to use this alternative flow (which isn't really an alternate).

This feedback is from the perspective that because we don't support older versions, and after the next release assume/expect users to be on the latest version, migration isn't possible so they have to follow the instructions in this section first.

I see the note at the top of the page and if you think that's good enough, all good. Just my 2c

Reply:
Yes I think the note covers it. I don't want to do too much with this vestigial text until we're farther down the line with new releases.